src/components/Form/Form.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from "react";
+import bem from "../../utils/bem";
+import generateUUID from "../../helpers/generateUUID";
+
+export interface FormProps {
+  /** Optional form `action` attribute */
+  action?: string;
+  /** Optional additional attributes passed to the `<form>` element */
+  attributes?: { [key: string]: any };
+  /** Optional className you can add in addition to `form` */
+  className?: string;
+  /** Optional ID that other components can cross reference */
+  id?: string;
+  /** Optional form `method` attribute */
+  method?: "get" | "post";
+  /** Optional modifiers array for use with BEM. See how to work with modifiers and BEM here: http://getbem.com/introduction/ */
+  modifiers?: string[];
+}
+
+// FormRow child-component
+export function FormRow({ ...props }) {
```

Comment:
For this line and line 27, you can replace this with `(props) {` since `{ ...props }` is the same as `props`.

Reply:
Done.